pkg/scheduler/queue/user_queues_test.go

```diff
@@ -487,15 +490,32 @@ func getUsersByQuerier(queues *queues, querierID string) []string {
 
 func TestShuffleQueriers(t *testing.T) {
 	allQueriers := []string{"a", "b", "c", "d", "e"}
+	tqs := tenantQuerierState{
+		querierIDsSorted: allQueriers,
+		tenantsByID: map[string]*queueTenant{
+			"team-a": {
+				shuffleShardSeed: 12345,
+			},
+		},
+		tenantQuerierIDs: map[string]map[string]struct{}{},
+	}
+
+	tqs.shuffleTenantQueriers("team-a", nil)
+	require.Nil(t, tqs.tenantQuerierIDs["team-a"])
 
-	require.Nil(t, shuffleQueriersForUser(12345, 10, allQueriers, nil))
```

Comment:
Would be good to keep this test case, where we check that setting `maxQueriers` larger than the number of available queriers returns `nil` (ie. "all queriers")

Reply:
done